classical/psr-large/domain.pddl

```diff
@@ -1,5 +1,5 @@
 (define (domain psr)
-  (:requirements :adl :derived-predicates)
+  (:requirements :equality :typing :derived-predicates :disjunctive-preconditions :negative-preconditions)
```

Comment:
`:derived-predicates` is a heavy hitter. Can you pull up all the domains that happen to have the requirement? We should double check that they do indeed need it.

Reply:
106: http://www.haz.ca/planning-domains/classical/philosophers/domain.pddl
48: http://www.haz.ca/planning-domains/classical/psr-middle/domain.pddl
41: http://www.haz.ca/planning-domains/classical/psr-large/domain.pddl
92: http://www.haz.ca/planning-domains/classical/optical-telegraphs/domain.pddl